src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,114 @@
-export const Pagination = () => {};
+import cn from 'classnames';
+import { PaginationRules } from '../../types/PaginationRules';
+
+interface Props<T> {
+  paginationRules: PaginationRules
+  items: T[]
+  onPageChanged: (rules: PaginationRules) => void
+}
+
+function createLinksList(amount: number,
+  activePage: number,
+  onClickCallBack: (pageNumber: number) => void) {
+  const linksList = [];
+
+  // eslint-disable-next-line no-plusplus
+  for (let i = 0; i < amount; i++) {
+    linksList.push(
+      <li className={cn('page-item', {
+        active: i === activePage - 1,
+      })}
+      >
+        <a
+          onClick={() => onClickCallBack(i + 1)}
+          data-cy="pageLink"
+          className="page-link"
+          href={`#${i + 1}`}
+          key={i}
+        >
+          {i + 1}
+        </a>
+      </li>,
+    );
+  }
+
+  return linksList;
+}
```

Comment:
Move to another file. 
Think it can be a helper function. ( you use jsx inside this function, best way return the array for mapping inside the component.

Can use Array.from(Array(amount).keys()), which creates an array from your amount. Then use array methods, and map the final result. 

You need to divide jsx and js in this func. Thanks.